plugins/org.jkiss.dbeaver.ext.mssql/src/org/jkiss/dbeaver/ext/mssql/model/SQLServerDialect.java

```diff
@@ -262,8 +262,12 @@ public String getColumnTypeModifiers(
     }
 
     @Override
-    public void generateStoredProcedureCall(StringBuilder sql, DBSProcedure proc, Collection<? extends DBSProcedureParameter> parameters,
-                                            boolean castParams) {
+    public void generateStoredProcedureCall(
+	    StringBuilder sql, 
+		DBSProcedure proc, 
+		Collection<? extends DBSProcedureParameter> parameters,
+        boolean castParams
```

Comment:
Don't mix tabs and spaces

```suggestion
        StringBuilder sql, 
        DBSProcedure proc, 
        Collection<? extends DBSProcedureParameter> parameters,
        boolean castParams
```